package.json

```diff
@@ -154,7 +154,7 @@
     "proxyquire": "~2.1.0",
     "semver": "~5.6.0",
     "should": "~13.2.3",
-    "sinon": "~2.3.7",
+    "sinon": "^7.0.0",
```

Comment:
Nitpick; let's change this to `~` so that we'll auto-update only patch versions, not minor versions.